webapp/src/pages/Settings.tsx

```diff
@@ -165,6 +177,56 @@ const displayReadonlyFields = (label: string, value: string | null) => (
     }}
   />
 );
+const SelectField: React.FC<
+  Omit<SelectProps, "onChange"> & {
+    label: string;
+    value: any;
+    onChange: (newValue: any) => void;
+    noneValue?: string;
```

Comment:
Why a string if we never display it? I guess a better name would be `nullable`.
```suggestion
    nullable?: boolean;
```

Reply:
That said, I don't think `spacy_model` is nullable, so we probably should just remove that.

Reply:
I resolved this in #494

Reply:
Thank you :)